docker-compose.yml

```diff
@@ -51,12 +51,16 @@ services:
       CROWN_API_PASSWORD: ${CROWN_API_PASSWORD:? CROWN_API_PASSWORD not set}
       CROWN_WITHDRAW_FEE_INT: ${CROWN_WITHDRAW_FEE_INT:? CROWN_WITHDRAW_FEE_INT not set}
       CROWN_WITHDRAW_NAME: ${CROWN_WITHDRAW_NAME:? CROWN_WITHDRAW_NAME not set}
+      MAIL_SERVER: ${MAIL_SERVER:? MAIL_SERVER not set}
```

Comment:
we can probably get rid of these because we control the mail server it will always have the same hostname/port (mail:587)

Reply:
we need this configuration to tell the app how to connect to the server.

its kinda like doing a telnet without knowing where the server and the port is.

Reply:
either do that or just hard code it in the email_utils.py

Reply:
statically placing in docker_compose.yml
```
MAIL_SERVER: mail
MAIL_POST: 587
```

and will run test to mkae sure it will try to connect to mail server.

Reply:
changed it first... will also change the app_core

Reply:
the above appears to work still after change.